mkdocs.yml

```diff
@@ -175,13 +174,7 @@ plugins:
             signature_crossrefs: true
             summary: true
             unwrap_annotated: true
-            # show_root_heading: true
```

Comment:
Turns out all these commented-out bits were already defined here, and with the same values. So can be safely removed.